beets/util/artresizer.py

```diff
@@ -209,7 +223,7 @@ def deinterlace(self, path_in, path_out=None):
             # FIXME: Should probably issue a warning?
             return path_in
 
-    def get_format(self, filepath):
+    def get_format(self, filepath: AnyStr) -> Optional[bytes]:
```

Comment:
Probably out-of scope here; but this code seems to be broken: All backends should really have the same return type, so we should probably be decoding the stdout to a `str` here.

Reply:
I actually intend to start my grand project to convert to pathlib today on the core modules, so this should be fixed soon regardless.